docs/administrator.md

```diff
@@ -103,18 +107,18 @@ operator with this RBAC policy use:
 
 Note that the service account in `operator-rbac.yaml` is named
 `zalando-postgres-operator`. You may have to change the `service_account_name`
-in the operator configmap and `serviceAccountName` in the postgres-operator
+in the operator ConfigMap and `serviceAccountName` in the postgres-operator
 deployment appropriately.
 
-This is done intentionally, as to avoid breaking those setups that already work
+This is done intentionally to avoid breaking those setups that already work
 with the default `operator` account. In the future the operator should ideally
 be run under the `zalando-postgres-operator` service account.
 
-The service account defined in  `operator-rbac.yaml` acquires some privileges
+The service account defined in `operator-rbac.yaml` acquires some privileges
 not really used by the operator (i.e. we only need list and watch on
-configmaps), this is also done intentionally to avoid breaking things if
+`configmaps`), this is also done intentionally to avoid breaking things if
```

Comment:
Shouldn't this also be 'ConfigMaps' instead?

Reply:
I kept in in lower case as it is referring to a value in one of the RBAC.yaml fields. I've also decorated the `list` and `watch` verbs so that the reference gets clearer